diesel/src/query_builder/upsert/on_conflict_clause.rs

```diff
@@ -95,7 +95,7 @@ where
     fn walk_ast<'b>(&'b self, mut out: AstPass<'_, 'b, DB>) -> QueryResult<()> {
         self.values.walk_ast(out.reborrow())?;
         out.push_sql(" ON DUPLICATE KEY");
-        self.target.walk_ast(out.reborrow())?;
+        // MySQL doesn't require a target
```

Comment:
Happy to take out this comment but leaving it for purposes of review now; odd to me but I suppose this keyword "ON DUPLICATE KEY" implicitly required for some kind of primary key- or unique-ness constraint on the table